_config.yml

```diff
@@ -143,6 +143,9 @@ plugins:
   - jekyll-twitter-plugin
   - jekyll-sitemap
   - jemoji
+  #if you want to uncomment next line, you have to set base_url beforehand
```

Comment:
how exactly should `base_url` be set? would be great to have this plugin enabled by default.

Reply:
Sorry, that was a typo. I meant `url` should be set. Something like: `url: pourmand1376.github.io`. Otherwise, it would throw an error. 

As the [site](https://github.com/keithmifsud/jekyll-target-blank) says:
> By default. all anchor tags and markdown links pointing to an external host, other than the one listed as the url in Jekyll's _config.yml will automatically be opened in a new browser tab once the site is generated.

So, if we want to make this default, I think we would have to make `url` it mandatory. That would also cause some compatibility issues. Does this sound right to you?

Reply:
I see, will have to investigate how to avoid compatibility issues. let's keep the PR open for the time being.